app/models/passenger.rb

```diff
@@ -0,0 +1,18 @@
+class Passenger < ApplicationRecord
+  has_many :trips
+
+  validates :name, presence: true 
+  validates :phone_num, presence: true, uniqueness: true
+
```

Comment:
It would be a good idea to have a `request_trip` method which generates a trip for an instance of `Passenger`